GitUI/CommandsDialogs/RevisionDiffControl.cs

```diff
@@ -757,6 +757,7 @@ private void UpdateStatusOfMenuItems()
             }
 
             DiffContextMenu.AddUserScripts(runScriptToolStripMenuItem, ExecuteCommand, script => script.OnEvent == ScriptEvent.ShowInFileList, UICommands);
+            toolStripSeparatorScript.Visible = runScriptToolStripMenuItem.HasDropDownItems;
```

Comment:
Scripts with `"OnEvent" == ShowInFileList` need to be considered, too.
Rather let `AddUserScript` return whether scripts have been added.